src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package duke.tasks;
```

Comment:
The order of your package statement and the class-level javadoc comment is inconsistent here. Given that most of your files have the comment before the package, consider shifting this package statement downwards?